components/tikv_util/src/metrics/mod.rs

```diff
@@ -113,3 +113,21 @@ pub fn convert_record_pairs(m: HashMap<String, u64>) -> RecordPairVec {
         })
         .collect()
 }
+
+pub trait StdThreadBuildWrapper {
```

Comment:
This is not an appropriate place to define the trait. Better put it in sys/thread.